Add `region_from_zone` provider function (#10073)
GoogleCloudPlatform/magic-modules: mmv1/third_party/terraform/website/docs/functions/region_from_zone.html.markdown

---
page_title: region_from_zone Function - terraform-provider-google
description: |-
  Returns the region within a provided zone.
---

# Function: region_from_zone

> Returns the region within a provided resource's zone.

Reviewer: On second reading of this, the phrasing sounds like it's saying the region is inside the zone. It means the region name substring within the zone name, but it could be confusing with a statement about regions belonging in zones.

```suggestion
Returns a region name derived from a provided zone.
```
